Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM opensuse/tumbleweed:latest
+
+# Install dependencies
+RUN zypper in -y -C \
+    git \
+    python3-osc \
+    python3-openqa_client \
+    python3-pika \
+    python3-requests \
+    python3-ruamel.yaml \
+    python3-jsonschema
+
+# Install application
+WORKDIR /app
+COPY . .
+# Alternatively, use sources from git
+# RUN git clone https://github.com/openSUSE/qem-bot .
```

Comment:
I suppose this `COPY` command is also where node modules are "installed" which is therefore an undocumented prerequisite. So just uncommenting the alternative to clone directly via Git is probably not going to work.

Reply:
Oh, right.